skimage/transform/tests/test_geometric.py

```diff
@@ -519,15 +519,15 @@ def test_degenerate():
     src = dst = np.zeros((10, 2))
 
     tform = SimilarityTransform()
-    tform.estimate(src, dst)
+    assert tform.estimate(src, dst)  # NOTE: Always returns true.
```

Comment:
Agree that this is correct for the current behavior, but it seems odd that it is inconsistent with the other transform types. Do you think we should change `estimate` to return False if any NaN are detected in the output of the `_umeyama` helper function?

Reply:
Sounds good.  Wasn't sure if it was appropriate to add here, but went ahead and added a commit for it.